RecoParticleFlow/PFClusterProducer/src/PFClusterEMEnergyCorrector.cc

```diff
@@ -14,81 +11,108 @@ namespace {
 }
 
 PFClusterEMEnergyCorrector::PFClusterEMEnergyCorrector(const edm::ParameterSet& conf, edm::ConsumesCollector &&cc) :
-  _calibrator(new PFEnergyCalibration) {
+  calibrator_(new PFEnergyCalibration) {
+
+
+   applyCrackCorrections_ = conf.getParameter<bool>("applyCrackCorrections");
+   applyMVACorrections_ = conf.getParameter<bool>("applyMVACorrections");
 
-   _applyCrackCorrections = conf.getParameter<bool>("applyCrackCorrections");
-   _applyMVACorrections = conf.getParameter<bool>("applyMVACorrections");
-   _maxPtForMVAEvaluation = conf.getParameter<double>("maxPtForMVAEvaluation");
+   maxPtForMVAEvaluation_ = conf.getParameter<double>("maxPtForMVAEvaluation");
   
+   ebSrFlagToken_ = cc.consumes<EBSrFlagCollection>(conf.getParameter<edm::InputTag>("ebSrFlagLabel"));
+   eeSrFlagToken_ = cc.consumes<EESrFlagCollection>(conf.getParameter<edm::InputTag>("eeSrFlagLabel"));
```

Comment:
these are not needed if srfAwareCorrection_ is false. Please put an ```if (srfAwareCorrection_) ``` around these consumes